java-client-model/src/main/java/com/cumulocity/rest/representation/inventory/ManagedObjects.java

```diff
@@ -0,0 +1,14 @@
+package com.cumulocity.rest.representation.inventory;
+
+import com.cumulocity.model.idtype.GId;
+
+public class ManagedObjects {
+    
+    public static ManagedObjectRepresentation asManagedObject(GId id) {
+        final ManagedObjectRepresentation managedObjectRepresentation = new ManagedObjectRepresentation();
+        managedObjectRepresentation.setId(id);
+        return managedObjectRepresentation;
+    }
```

Comment:
we can reconsider if such util class with single method is needed or better to move this method to ManagedObject

Reply:
Make sense, but this method has more then 50 usages - I did't want this PR to blow up completely. I will address it in separate PR
TODO